extension-manifest-v3/src/pages/onboarding/index.js

```diff
@@ -19,10 +19,9 @@ async function updateOptions(host, event) {
   const success = event.type === 'success';
 
   await store.set(Options, {
-    engines: Object.keys(Options.engines).reduce(
-      (all, rule) => ({ ...all, [rule]: success }),
-      {},
-    ),
+    ads: success,
```

Comment:
how about we make those names more description, eg. `protectionAds` or `featureAdBlocking`, or maybe `Ad-Blocking`, `Anti-Tracking` `Never-Consent`?

Reply:
`feature` prefix - all options are a kind of feature. I would avoid using dashes in names, as you are forced to use `options["Ad-Blocking"] in the code.

The `engines` scope was fine. The only reason to move it to the top is this sync feature. If there is a need for the indicator, might be something, which is related to the logic, like `block` ? `blockAds`, `blockTrackers`, `blockAnnoyances` ?